src/wrapper/manual_commons.jl

```diff
@@ -0,0 +1,46 @@
+# TODO: created manually, better to improve on wrapper generator!
```

Comment:
Where are these values defined in SCIP? What went wrong with the wrapper generator?

Reply:
These are defined in the SCIP headers that I wrap, but for some reason (did not investigate yet) were skipped by Clang.jl.

The `FILE` type is just standard C, I guess? It's not defined by SCIP itself, at least.

Reply:
see #77